sklearn/covariance/_graph_lasso.py

```diff
@@ -168,7 +168,7 @@ def graphical_lasso(emp_cov, alpha, *, cov_init=None, mode='cd', tol=1e-4,
     _, n_features = emp_cov.shape
     if alpha == 0:
         if return_costs:
-            precision_ = linalg.inv(emp_cov)
+            precision_ = linalg.inv(emp_cov, check_finite=False)
```

Comment:
In this case since `emp_cov` was not checked before, we would still require `linalg.inv` to perform the check.

Reply:
let me remove it for this particular case and change it back to `precision_ = linalg.inv(emp_cov)`